bin/i18n/sync-out.rb

```diff
@@ -268,4 +269,25 @@ def rebuild_blockly_js_files
   end
 end
 
+# In the sync in, we slice the YAML headers of the files we upload to crowdin
+# down to just the part we want to translate (ie, the title). Here, we
+# reinflate the header with all the values from the source file.
+def restore_markdown_headers
```

Comment:
has this been moved from somewhere, or just copied?

Reply:
Adapted from here: https://github.com/code-dot-org/code-dot-org/blob/5eb122c96bca660c9b8fd430bdf8c1e27eac7aac/bin/i18n/hoc_sync_utils.rb#L108

I thought about trying to combine them but there's just enough differences that they would need some refactoring, plus most of the calls are to shared functions anyway.